influxdb/content.md

```diff
@@ -1,194 +1,138 @@
-$ docker run -d -p 8086:8086 \
-      -v $PWD/data:/var/lib/influxdb2 \
-      -v $PWD/config:/etc/influxdb2 \
-      -e DOCKER_INFLUXDB_INIT_MODE=setup \
-      -e DOCKER_INFLUXDB_INIT_USERNAME=my-user \
-      -e DOCKER_INFLUXDB_INIT_PASSWORD=my-password \
-      -e DOCKER_INFLUXDB_INIT_ORG=my-org \
-      -e DOCKER_INFLUXDB_INIT_BUCKET=my-bucket \
-      %%IMAGE%%:2.0
+docker run -d -p 8086:8086 \
+    -v $PWD/data:/var/lib/influxdb2 \
+    -v $PWD/config:/etc/influxdb2 \
+    -e DOCKER_INFLUXDB_INIT_MODE=setup \
+    -e DOCKER_INFLUXDB_INIT_USERNAME=my-user \
+    -e DOCKER_INFLUXDB_INIT_PASSWORD=my-password \
+    -e DOCKER_INFLUXDB_INIT_ORG=my-org \
+    -e DOCKER_INFLUXDB_INIT_BUCKET=my-bucket \
+    -e DOCKER_INFLUXDB_INIT_RETENTION=1w \
+    -e DOCKER_INFLUXDB_INIT_ADMIN_TOKEN=my-super-secret-auth-token \
+    %%IMAGE%%:2
 ```
 
-And an example using all available options is:
+*To upgrade from InfluxDB 1.x to InfluxDB 2.x, see the **Upgrading from InfluxDB 1.x** section below.\*
 
-```console
-$ docker run -d -p 8086:8086 \
-      -v $PWD/data:/var/lib/influxdb2 \
-      -v $PWD/config:/etc/influxdb2 \
-      -e DOCKER_INFLUXDB_INIT_MODE=setup \
-      -e DOCKER_INFLUXDB_INIT_USERNAME=my-user \
-      -e DOCKER_INFLUXDB_INIT_PASSWORD=my-password \
-      -e DOCKER_INFLUXDB_INIT_ORG=my-org \
-      -e DOCKER_INFLUXDB_INIT_BUCKET=my-bucket \
-      -e DOCKER_INFLUXDB_INIT_RETENTION=1w \
-      -e DOCKER_INFLUXDB_INIT_ADMIN_TOKEN=my-super-secret-auth-token \
-      %%IMAGE%%:2.0
-```
+With InfluxDB set up and running, see the [Get started](https://docs.influxdata.com/influxdb/v2/get-started/) tutorial to create tokens and write and query data.
 
-**NOTE:** Automated setup will not run if an existing boltdb file is found at the configured path. This behavior allows for the InfluxDB container to reboot post-setup without encountering "DB is already set up" errors.
+### Custom Initialization Scripts
 
-### Interacting with InfluxDB
+In `setup` mode (`DOCKER_INFLUXDB_INIT_MODE=setup`) or `upgrade` mode (`DOCKER_INFLUXDB_INIT_MODE=upgrade`), the InfluxDB Docker Hub image supports running custom initialization scripts. After the setup process completes, scripts are executed in lexical sort order by name.
 
-Once an InfluxDB instance has completed initial setup, its APIs will unlock. See the main documentation site for reference information and examples on:
+For the container to run scripts, they must:
 
--	[Writing data](https://docs.influxdata.com/influxdb/v2.0/write-data/)
--	[Reading data](https://docs.influxdata.com/influxdb/v2.0/query-data/)
--	[Configuring security](https://docs.influxdata.com/influxdb/v2.0/security/)
--	[And more!](https://docs.influxdata.com/influxdb/v2.0/)
+-	Be mounted in the container's `/docker-entrypoint-initdb.d` directory
+-	Be named using the `.sh` file name extension
+-	Be executable by the user running the `docker run` command--for example, to allow the current use to execute a script with `docker run`:
```

Comment:
Do these tabs affect the markdown rendering? I generally use spaces for indentation.

Reply:
Same. I replaced them all only to find that tabs are required by their linter (markdownfmt).